bin/generate-settings-manifest

```diff
@@ -3,18 +3,10 @@
 # DOC: Generates the SettingsManifest.java file based on the contents of env_var_docs.json
 
 source bin/lib.sh
+docker::set_project_name_env_var_docs
+docker::compose_env_var_docs_up
 
-if ! python::env_var_docs_venv_is_installed; then
-  echo "env var docs venv not found, creating..."
-  bin/env-var-docs-create-venv
-fi
-
-if ! python::env_var_docs_venv_is_active; then
-  echo "env var docs venv not activated, activating (this command only)..."
-  source env-var-docs/venv/bin/activate
-fi
-
-ENV_VAR_DOCS_PATH=server/conf/env-var-docs.json \
-  python3 env-var-docs/parser-package/src/env_var_docs/settings_manifest.py
+LOCAL_OUTPUT=false
+docker::run_env_var_docs_command python env-var-docs/parser-package/src/env_var_docs/settings_manifest.py
```

Comment:
**nitpick:** putting `LOCAL_OUTPUT=false` on the same line as the call to `docker::run_env_var_docs_command` will keep people from wondering if it's consumed somewhere further down as well